sencha-workspace/SlateTasksTeacher/app/controller/Dashboard.js

```diff
@@ -394,13 +415,37 @@ Ext.define('SlateTasksTeacher.controller.Dashboard', {
                 SectionID: me.getCourseSelector().getSelection().getId()
             });
 
-        me.doSaveStudentTask(studentTask, function() {
-            taskAssigner.close();
-            me.getStudentTasksStore().load({
-                id: studentTask.getId(),
-                addRecords: true
-            });
-            Ext.toast(student.getFullName() + ' was assigned task: ' + task.get('Title'));
+        me.doSaveStudentTask(studentTask, function(rec, success, request) {
+            var response,
+                message = [],
+                validationErrors, key;
+
+            if (success) {
+                taskAssigner.close();
+                me.getStudentTasksStore().load({
+                    id: studentTask.getId(),
+                    addRecords: true
+                });
+                Ext.toast(student.getFullName() + ' was assigned task: ' + task.get('Title'));
+            } else {
+                response = JSON.parse(request.getResponse().responseText);
+                message.push(
+                    'Failed to assign',
+                    '<b>'+rec.get('Task').Title+'</b>',
```

Comment:
@nbey use `<strong>` instead of `<b>` and wrap the message in a `<p>` instead of stacking `<br>`s to do the same thing